libkbfs/journal_server.go

```diff
@@ -301,6 +455,11 @@ func (j *JournalServer) Disable(ctx context.Context, tlfID TlfID) (err error) {
 		return fmt.Errorf("Journal still has %d MD entries", length)
 	}
 
+	select {
+	case bundle.shutdownCh <- struct{}{}:
```

Comment:
For the same reason as above, multiple shutdown calls get coalesced into one, which is fine. If I close here, then that's problematic since subsequent closes will panic.